k8s/helmfile/prometheus-operator.yaml

```diff
@@ -0,0 +1,6 @@
+releases:
+  - name: prometheus-operator-crds
+    namespace: monitoring
```

Comment:
it's probably fine and desired but trying to apply this to a cluster that already had an earlier version of this PR on it results in:
```
Error: Failed to render chart: exit status 1: Error: rendered manifests contain a resource that already exists. Unable to continue with install: CustomResourceDefinition "alertmanagerconfigs.monitoring.coreos.com" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-namespace" must equal "monitoring": current value is "default"
  Error: plugin "diff" exited with error
```

Reply:
So what I did just this morning was:

- delete local cluster
- create new cluster from scratch
- apply main
- checkout this branch
- apply this branch

which passed for me without any issues. Not sure why yours is now complaining about duplicate CRDs?